context/build.gradle

```diff
@@ -0,0 +1,16 @@
+plugins {
+    id "be.insaneprogramming.gradle.animalsniffer" version "1.4.0"
+}
+
+description = 'gRPC: Context'
+
+dependencies {
+    compile libraries.guava,
```

Comment:
I'm very excited about this. It would be great to share a context for
zipkin, grpc and census (and whatever else)

On Fri, Sep 2, 2016 at 8:46 AM, Eric Anderson <notifications@github.com>
wrote:

> In context/build.gradle
> <https://github.com/grpc/grpc-java/pull/2226#discussion_r77278670>:
>
> > @@ -0,0 +1,16 @@
> > +plugins {
> > +    id "be.insaneprogramming.gradle.animalsniffer" version "1.4.0"
> > +}
> > +
> > +description = 'gRPC: Context'
> > +
> > +dependencies {
> > +    compile libraries.guava,
>
> @adriancole <https://github.com/adriancole>, I didn't say anything during
> review, but I was already thinking we'd do a follow-up to remove the dep
> :). Now works, too.
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub
> <https://github.com/grpc/grpc-java/pull/2226/files/d41fa3dce7f0a88c99267604b55105dc5916f980#r77278670>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/AAD610heCwL44cNXRTKtd69gInW3vCDCks5ql3HJgaJpZM4JzN71>
> .
>